src/responder/kcm/secrets/secrets.c

```diff
@@ -692,15 +691,6 @@ errno_t sss_sec_new_req(TALLOC_CTX *mem_ctx,
     }
     req->sctx = sec_ctx;
 
-    /* drop the prefix and select a basedn instead */
-    if (geteuid() != KCM_PEER_UID && client != KCM_PEER_UID) {
```

Comment:
KCM_PEER_UID is now unused, you can remove it.

Reply:
Done.